frontend/src/app/workflows/model/examples/ms.example.ts

```diff
@@ -0,0 +1,353 @@
+export const msExample =
+  '- workflow: smooth case\n' +
+  '\n' +
+  '- user: Peter Picker\n' +
```

Comment:
You could use multiline strings with backticks here. Alternatively, I prefer to put example files as assets.

Reply:
I will never forget multiline string ever again